### DISCUSSION TITLE
State of interprocess

### DISCUSSION THREAD

**Discussion by ottosson** [kotauskas/interprocess#34] [General]
Hi!

I'm shopping around for a ipc solution and came across this project. I've seen some activity recently and was just wondering if you have any idea when you will be close to the 2.0 version.

Also, do you feel it's a good idea to use the 1.x until then?

Thanks!

**Comment by kotauskas** [comment]
I still have a few things to sort out before I consider the 2.0.0 release entirely polished and up to my standard for it, but it's fairly close to release – the only thing holding me back is the inconsistent amount of time I end up allocating to it, since I do it in my free time, when I have that free time to begin with.

That being said, 1.2.1 is a very usable release with only a few edge-case bugs, performance hiccups and API deficiencies, mostly surrounding Tokio support. If you're comfortable with this state of affairs, you can depend on the non-async functionality of 1.2.1 for a prototype of your own program and upgrade incrementally to 2.0.0 for its full release.

**Reply by ottosson** [reply]
I totally understand! I appreciate the work you are doing.

I will be using Tokio quite a bit in my application. Maybe it's best to wait until you get the motivation to finish 2.0 then.
I would probably be just fine with 1.2.1 but when something new is on the horizon it's hard not to wait for that =P

Thanks!

**Reply by kotauskas** [reply]
@ottosson, forgot to notify you here – I finally released 2.0.0 a few weeks ago, with 2.1.0 being the most recent version containing a few more enhancements.
